### PR TITLE
commit message

### PR DESCRIPTION
**Pull request by olya-yevsieienko**
- [DEMO LINK](https://olya-yevsieienko.github.io/layout_hello-world/)
- [TEST REPORT LINK](https://olya-yevsieienko.github.io/layout_hello-world/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+    Hello, world!
+  </body>
+</html>
```
Comment (illiarerun) [inline src/index.html:12]: Always add a blank line at the end of your files with any code. 
```suggestion
</html>

```

File: .linthtmlrc.json [hunk 2]
```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
```
Comment (illiarerun) [inline .linthtmlrc.json:2]: do not change linter settings, this file should not be edited

**Review by illiarerun — CHANGES_REQUESTED** [review]
Nice start, now let's work on fixes:

**Comment by olya-yevsieienko** [comment]
Thanks, I changed it!

ср, 27 апр. 2022 г. в 12:29, Illia Vovchenko ***@***.***>:

**Review by illiarerun — APPROVED** [review]
Good job 🔥 Btw, about blank line, I think I've missed that you already had one :harold:

**Pull request closed without merging**